scripts/pre_commit_linter.py

```diff
@@ -2178,6 +2190,23 @@ def _check_codeowner_file(self):
                             print ('WARNING! %s/%s is not covered under '
                                    'CODEOWNERS' % (root, file_name))
 
+            # Checks that the most important rules/patterns are at the bottom
+            # of the CODEOWNERS file.
+            important_path_match_bool_list = ([
+                imp_path in path_lines[-1 * len(
+                    CODEOWNER_IMPORTANT_PATHS):]
+                for imp_path in CODEOWNER_IMPORTANT_PATHS])
+            if not all(important_path_match_bool_list):
+                for index, bool_value in enumerate(
+                        important_path_match_bool_list):
+                    if not bool_value:
+                        print ('%s --> Please ensure that pattern \'%s\' is'
```

Comment:
Unclear error message. Please re-frame to something like:
`Please ensure that the rule <pattern> lies towards the bottom of the CODEOWNERS file here <codeowner_filepath> since it is an important rule.`

Reply:
Done!